torch/_inductor/lowering.py

```diff
@@ -1222,6 +1222,210 @@ def apply_constraint(arg, fx_arg):
+make_fallback(aten.special_chebyshev_polynomial_u)
+make_fallback(aten.special_entr)
+make_fallback(aten.special_erfcx)
+make_fallback(aten.special_hermite_polynomial_h)
+make_fallback(aten.special_hermite_polynomial_he)
+make_fallback(aten.special_i0e)
+make_fallback(aten.special_i1)
+make_fallback(aten.special_i1e)
+make_fallback(aten.special_laguerre_polynomial_l)
+make_fallback(aten.special_log_ndtr)
+make_fallback(aten.special_modified_bessel_i0)
+make_fallback(aten.special_modified_bessel_i1)
+make_fallback(aten.special_modified_bessel_k0)
+make_fallback(aten.special_modified_bessel_k1)
+make_fallback(aten.special_ndtri)
+make_fallback(aten.special_scaled_modified_bessel_k0)
+make_fallback(aten.special_scaled_modified_bessel_k1)
+make_fallback(aten.special_spherical_bessel_j0)
+make_fallback(aten.special_xlog1py)
+make_fallback(aten.special_zeta)
+make_fallback(aten.take)
+make_fallback(aten.threshold)
+make_fallback(aten.trace)
+make_fallback(aten._trilinear)
+make_fallback(aten.unfold_copy)
+make_fallback(aten.unsafe_split)
+make_fallback(aten.vdot)
+make_fallback(aten.view_as_complex)
+make_fallback(aten.view_copy)
+make_fallback(aten.xlogy)
+make_fallback(aten._adaptive_avg_pool3d_backward)
+make_fallback(aten.adaptive_max_pool2d_backward)
+make_fallback(aten.adaptive_max_pool3d_backward)
+make_fallback(aten.avg_pool3d_backward)
+make_fallback(aten.binary_cross_entropy_backward)
+make_fallback(aten.bitwise_or_)
+make_fallback(aten._cdist_backward)
+make_fallback(aten.diagonal_backward)
+make_fallback(aten._embedding_bag_dense_backward)
+make_fallback(aten.fractional_max_pool2d_backward)
+make_fallback(aten.fractional_max_pool3d_backward)
+make_fallback(aten.hardshrink_backward)
+make_fallback(aten.huber_loss_backward)
+make_fallback(aten._linalg_check_errors)
+make_fallback(aten.log_sigmoid_backward)
+make_fallback(aten.max_pool3d_with_indices_backward)
+make_fallback(aten.multilabel_margin_loss_backward)
+make_fallback(aten.multi_margin_loss_backward)
+make_fallback(aten._pdist_backward)
+make_fallback(aten._prelu_kernel_backward)
+make_fallback(aten.reflection_pad1d_backward)
+make_fallback(aten.replication_pad1d_backward)
+make_fallback(aten.smooth_l1_loss_backward)
+make_fallback(aten.soft_margin_loss_backward)
+make_fallback(aten.softshrink_backward)
+make_fallback(aten.squeeze_copy)
+make_fallback(aten.linalg_pinv.atol_rtol_tensor)
+make_fallback(aten.segment_reduce.default)
+make_fallback(aten._segment_reduce_backward.default)
+make_fallback(aten.angle)
+make_fallback(aten.cholesky_inverse)
+make_fallback(aten.cholesky_solve)
+make_fallback(aten._fft_r2c)
+make_fallback(aten.histogram.bin_ct)
+make_fallback(aten._histogramdd_bin_edges.default)
+make_fallback(aten._histogramdd_from_bin_cts.default)
+make_fallback(aten.index_reduce)
+make_fallback(aten.masked_scatter)
+make_fallback(aten.to_sparse)
+make_fallback(aten.triangular_solve)
+make_fallback(aten.expand_copy)
+make_fallback(aten.zeros)
+make_fallback(aten.gcd.default)
```

Comment:
I see lots of simple pointwise ops that could be supported with a few lines of code and also variants of things already supported here. We should just add support for the easy ones.

Reply:
may be better to do that in a follow-up PR in this stack not to clutter this one too much.

Reply:
Yes, I'm working on this. I was going to do it as a second PR in same `ghstack` stack but LMK if you think it's better to do all in same PR.